src/cli/fuzzer_bridge/models.py

```diff
@@ -0,0 +1,140 @@
+"""Pydantic models for fuzzer output format v2."""
+
+from typing import Dict, List, Optional, Union
+
+from pydantic import BaseModel, Field, field_validator
+
+from ethereum_test_base_types import Address, Bytes, Hash, HexNumber
+
+# ruff: noqa: N815  # Allow mixedCase for JSON field compatibility
+
+
+class FuzzerAccount(BaseModel):
+    """Account definition in fuzzer output."""
+
+    balance: HexNumber
+    nonce: HexNumber = HexNumber(0)
+    code: Bytes = Bytes(b"")
+    storage: Dict[HexNumber, HexNumber] = Field(default_factory=dict)
+    privateKey: Optional[Bytes] = Field(None, alias="privateKey")
+
+    @field_validator("storage", mode="before")
+    @classmethod
+    def validate_storage(cls, v):
+        """Convert storage keys and values to HexNumber."""
+        if not v:
+            return {}
+        return {HexNumber(k): HexNumber(v_) for k, v_ in v.items()}
+
+
+class FuzzerAuthorization(BaseModel):
+    """Authorization tuple for EIP-7702 SetCode transactions."""
+
+    chainId: HexNumber = Field(..., alias="chainId")
+    address: Address
+    nonce: HexNumber
+    v: HexNumber  # yParity
+    r: HexNumber
+    s: HexNumber
+
+    class Config:
+        """Pydantic configuration."""
+
+        populate_by_name = True
+
+
+class FuzzerTransaction(BaseModel):
+    """Transaction definition in fuzzer output."""
+
+    from_: Address = Field(..., alias="from")
+    to: Optional[Address] = None
+    value: HexNumber = HexNumber(0)
+    gas: HexNumber
+    gasPrice: Optional[HexNumber] = Field(None, alias="gasPrice")
+    maxFeePerGas: Optional[HexNumber] = Field(None, alias="maxFeePerGas")
+    maxPriorityFeePerGas: Optional[HexNumber] = Field(None, alias="maxPriorityFeePerGas")
+    nonce: HexNumber
+    data: Bytes = Bytes(b"")
+    accessList: Optional[List[Dict[str, Union[str, List[str]]]]] = Field(None, alias="accessList")
+    blobVersionedHashes: Optional[List[Hash]] = Field(None, alias="blobVersionedHashes")
+    maxFeePerBlobGas: Optional[HexNumber] = Field(None, alias="maxFeePerBlobGas")
+    authorizationList: Optional[List[FuzzerAuthorization]] = Field(None, alias="authorizationList")
+
+    class Config:
+        """Pydantic configuration."""
+
+        populate_by_name = True
```

Comment:
This is basically `from ethereum_test_types import Transaction`, so we can reduce this model to:
```suggestion
class FuzzerTransaction(Transaction):
    """Transaction definition in fuzzer output."""

    from_: Address = Field(..., alias="from")
```